docs/contributing/gatsby-style-guide.md

```diff
@@ -404,6 +404,17 @@ wording.
 
 ## Best practices
 
+### Referencing Gatsby Cloud
+
+While Gatsby Cloud is hosted on a separate site [gatsbyjs.com](https://www.gatsbyjs.com/), it is part of the Gatsby [founding organization](/blog/2018-05-24-launching-new-gatsby-company/) and focused specifically on Gatsby sites. There are various parts of the OSS documentation that may benefit from pointing to Gatsby Cloud as a potential platform to explore.
+
+The guidelines for doing so are as follows:
+
+- If possible, Gatsby Cloud should be accompanied by other relevant technologies
+- If Gatsby Cloud does something by default, the docs should still include instructions for accessing that functionality manually
+
+The spirit of these guidelines is to ensure that users are aware of multiple options for running their Gatsby site. However, the open source documentation should preclude assumptions about technology choices other than gatsby-cli.
```

Comment:
```suggestion
The spirit of these guidelines is to ensure that users are aware of multiple options for running their Gatsby site. However, the open source documentation should preclude assumptions about technology choices other than Gatsby Cloud.
```

Maybe I'm missing something but I'm assuming this is what you meant?

Reply:
No I meant gatsby-cli. But perhaps this means I'm being unclear. My goal here is to say that OSS docs are unopinionated and the only requirement within them should be the gatsby CLI.

Reply:
Ahhhh I see. Maybe...
```suggestion
The spirit of these guidelines is to ensure that users are aware of multiple options for running their Gatsby site. With the exception of gatsby-cli, the open source documentation should generally preclude assumptions about technology choices.
```